examples/all.yaml

```
@@ -258,3 +258,12 @@ fio_git_username: ""
 fio_git_token: ""
 fio_git_branch: ""
 fio_cleanup: true
+
+#cron-job vars
+enable_cronjob: false  # Set true to enable the cronjob
+cronJob_schedule: "* * * * *"
```

refrain from the use of camel case to maintain consistency with other playbooks.
```suggestion
cronjob_schedule: "* * * * *"
```

---

updated the variable name